src/main/main.ts

```diff
@@ -738,6 +738,12 @@ function registerAllIpcListeners() {
                 .catch((err) => logger.error(err));
         }
     });
+
+    ipcMain.on(IpcChannels.executeUrl, (event, userInput: string) => {
+        openUrlInBrowser(`${config.urlOptions.defaultProtocol}://${userInput}.com`)
+            .then(() => { /* do nothing */ })
+            .catch((err) => logger.error(err));
+    })
```

Comment:
This should be handled by the UrlPlugin.

Reply:
But I can't access the UrlPlugin directly, maybe extract the logic of making a url search result item into a helper function ?